exercises/D-logical-operators/exercise2.js

```diff
@@ -4,8 +4,36 @@
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
 */
+// need to put a name in the function
+// name num
+
+
+function isNegative(num) {
+  return num < 0;
+}
+
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+      return true;
+  } 
+}
+
+function isShortName(name) {
```

Comment:
All correct solutions, really good work! Not essential but a cleaner solution for the isShortName and startsWithD functions would be to use the ternary operator, if you're curious here's the docs for it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator 

Let me know if you have any related questions